x-pack/solutions/security/plugins/security_solution/server/endpoint/schemas/artifacts/lists.ts

```diff
@@ -123,12 +145,13 @@ export const translatedPerformantEntries = t.array(
 
 export type TranslatedPerformantEntries = t.TypeOf<typeof translatedPerformantEntries>;
 
-export const translatedExceptionListItem = t.exact(
+export const translatedExceptionListItem = t.union([
   t.type({
     type: t.string,
     entries: t.array(translatedEntry),
-  })
-);
+  }),
+  translatedEntryTrustDescendants,
+]);
```

Comment:
The first union member lost its `t.exact()` wrapper when converting to a union type. Was this intentional?
Maybe it should be 

```typescript
export const translatedExceptionListItem = t.union([
  t.exact(t.type({
    type: t.string,
    entries: t.array(translatedEntry),
  })),
  translatedEntryTrustDescendants,
]);
```